pep/app/services/printer.ts

```diff
@@ -11,13 +26,27 @@ export default class PrinterService extends Service {
      * @param {{ field: keyof M; displayName: string }[]} properties
      * @memberof PrinterService
      */
-    print<M>(data: M[], properties: { field: keyof M; displayName: string }[]) {
+    printJSON<M>(data: M[], properties: { field: keyof M; displayName: string }[]) {
         printJS({
             printable: data,
             type: 'json',
             properties
         });
     }
+
+    /**
+     * Print data using [PrintJS](https://printjs.crabbly.com/)
+     *
+     * @param {string} data
+     * @memberof PrinterService
+     */
+    printHTML(data: string) {
+        printJS({
+            printable: data,
+            type: 'raw-html',
+            style: `.list-unstyled { padding-left: 0; list-style: none; font-size: 12px; } .mb-3 { margin-bottom: 1rem !important; }`
```

Comment:
maybe not worth it yet, but if we ever have to add/duplicate more styles to this, we should consider just doing a fetch() of the `/vendor.scss` as text and throw it in here, so you have all the bootstrap styles at your disposable. and maybe even cache the styles in this service so it only needs to be loaded once per session.

Reply:
I actually initially just imported the pep and vendor.css but then the paging didn't work. I didn't want to spend hours trying to figure out what broke it so I went this way and just included what I needed

Reply:
whats the paging? But yeah loading the `pep.css` might've been the culprit (or part of it) since it has a bunch of global layout styles and stuff that probably would muck with print display majorly.

Though i just realized that we import Bootstrap in our app (pep.css) file, so just using the vendor.css only wouldn't work anyway.. we'd need to think of a way to pull that apart, or maybe just add some global print styles that override the stuff that is causing it to break in print.